python/lsst/meas/base/classification.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008-2016 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+"""
+Definition and registration of classification plugins
+"""
+
+import numpy
+
+import lsst.pex.config
+from .afterburner import AfterburnerPluginConfig, AfterburnerPlugin
+from .pluginRegistry import register
+
+__all__ = (
+    "ABClassificationConfig", "ABClassificationPlugin",
+)
+
+
+class ABClassificationConfig(AfterburnerPluginConfig):
+
```

Comment:
Remove the blank line since it doesn't add anything and adds potential for the parts being separated.